docs/sc-tools/codegen_removal.md

```diff
@@ -0,0 +1,12 @@
+# codegen-removal
+
+This is a program that translates sc-machine usage from old format into new format.
```

Comment:
```suggestion
This is a program that translates all agents implemented on the old API supported on sc-machine versions <= 0.9.0 to the new API.
```

Reply:
not only agents are translated, I added list with everything that is translated